llvm/test/MC/ELF/warn-newline-in-escaped-string.s

```diff
@@ -0,0 +1,50 @@
+// RUN: llvm-mc -filetype=obj -triple x86_64 %s 2>&1 -o /dev/null | FileCheck -DFILE=%s --strict-whitespace %s --implicit-check-not=valid1_string --implicit-check-not=valid2_string --implicit-check-not=valid3_string --check-prefix=CHECK-WARN
```

Comment:
There should be just one --implicit-check-not with a regex for the undesired warning. This will work as long as all expected warnings are checked by CHECK lines.

Reply:
like `--implicit-check-not=valid*_string` ?

Reply:
E.g. `--implicit-check-not="{{[0-9]+:[0-9]+: warning: unterminated string}}"`

Reply:
Thanks, got it

Reply:
Done